readme.md

```diff
@@ -414,6 +424,16 @@
 | ------: | -------: | -----: |
 |   2 - 4 |       13 | 30-45m |
 
+### [Heroscape](https://boardgamegeek.com/boardgame/11170/heroscape-master-set-rise-valkyrie)
+
+> Join the battle of all time! Draft your army from troops of all different eras, species, and battle styles then fight on a battlefield created by you and your friends. Work together to accomplish a scenario laid out in the included rules or duke it out amongst yourselves to see who's the greater battlefield commander.
+
+![Heroscape Master Set: Rise of the Valkyrie](https://cdn.shopify.com/s/files/1/1031/9405/products/1_13261260-646a-4892-a1ab-737aa730c902.jpeg?v=1589744301)
```

Comment:
```suggestion
![Heroscape Master Set: Rise of the Valkyrie](https://cf.geekdo-images.com/itemrep/img/QxW14xwSXjhYU8h_KlvG09nbBd0=/fit-in/246x300/filters:strip_icc()/pic244662.jpg)
```

Similar story to this one. Normally I'd like people to keep PRs for games separate, but it's fine.

Reply:
This image hasn’t changed and its current resolution is still a bit over size (400x300), with a fair amount of white space. The suggested code change image source is 246x125 and has no extra white space in the image.